src/config/config.rs

```diff
@@ -109,14 +110,26 @@ impl Config {
             }
         };
 
-        let girs_dir: PathBuf = match girs_dir.into() {
-            Some("") | None => {
-                let path = toml.lookup_str("options.girs_dir", "No options.girs_dir")?;
-                config_dir.join(path)
+        let mut girs_dirs: Vec<PathBuf> = girs_dirs
```

Comment:
Why not leave `git_dir` as shortcut for single item list?

Reply:
I think we shouldn't have both options because it tends to make think that the other doesn't exist.